src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java

```diff
@@ -55,9 +55,9 @@ public ForumBoardPage open(String forumBoardTitle) {
   public String createNew(User user) {
     String content = String.format("%s%d", PageContent.ARTICLE_TEXT, DateTime.now().getMillis());
     String boardName = String.format("%s%d", "ForumBoard", DateTime.now().getMillis());
+    String boardTitle = String.format("%s:%s", URLsContent.FORUM_BOARD_NAMESPACE, boardName);
```

Comment:
Maybe move adding `FORUM_BORD_NAMESPACE` from here to `ForumBoardContent`?

Reply:
Same for templates?

Reply:
Done in https://github.com/Wikia/selenium-tests/pull/1943/commits/bb890f5ffde02e90fdfece37e3b39a2eb11555c5 for ForumBoardContent. It looks like that's already done for Templates. :)